fabric/src/generated/resources/data/c/tags/items/material/blackstone.json

```diff
@@ -0,0 +1,14 @@
+{
+  "replace": false,
+  "values": [
+    "#consistency_plus:material/blackstone",
+    "minecraft:polished_blackstone",
+    "minecraft:blackstone_slab",
+    "minecraft:blackstone_stairs",
+    "minecraft:blackstone",
+    "minecraft:blackstone_wall",
+    "minecraft:polished_blackstone_wall",
+    "minecraft:polished_blackstone_slab",
```

Comment:
missing vanilla's blackstone bricks due to "polished_blackstone_bricks" being the ID